pyproject.toml

```diff
@@ -42,7 +42,7 @@ python-can = "4.0.0"
 bitstruct = "6.0.0"
 Twisted = "21.2.0"
 qt5reactor = "0.5"
-pyelftools = "0.26"
+pyelftools = { git = "https://github.com/eliben/pyelftools", rev = "27941c50fef8cff8ef991419511664154c8cdf52" }
```

Comment:
Why use the git hash instead of the version ".25"?

you changed it to .25 in stlib so why make it different here?

Reply:
Using the hash for `st` and `stlib`. 1aea665cdb5613303920428b1993d4e320a660f9